### ISSUE TITLE
Can't get sync to work

### ISSUE THREAD

**Issue by chaapp87** [crschnick/pdx_unlimiter#79]
Hi there,

I'm trying to sync my savegames between 2 pcs running pdx-unlimiter.
What would be the best way?
I tried using the Sync/Backup Storage feature. But I don't get this to work.

I'd like to sync files automatically via gdrive.

Can anybody help?

Many thanks in advance!

**Comment by crschnick** [comment]
The Sync/Backup Storage feature is mainly intended to be used to maintain external backups of your savegames or migrate them to another system.

In case you are running pdxu on two systems intertwined, I would recommend to do the following:
- In your gdrive folder on your pc, create a new folder like `<Google Drive>\pdxu savegames`
- In the Pdx-Unlimiter settings menu, set the `storage directory` to this newly created folder

This should automatically sync all changes with your Google Drive.

**Comment by crschnick** [comment]
One problem that might happen is that pdxu will complain that the folder is not empty when doing this on your second system because Google Drive has already synced the files on that system.

So you maybe should start out with an empty folder to make this work properly.

**Comment by chaapp87** [comment]
OK thanks, that worked.
First I created a backup of my savegames.
Then I created an empty dir in GDrive.
Afterwards I configured PDX Unlimter on the first machine to use this folder for savegames.
It then "moved" savegames to the folder. I then closed the tool and deleted the folder content.
Then I did the same on the second system and deleted the folder content again.
I then copied my backup into the folder.
On the first computer I imported the savegames and closed the app. After the sync was done, I could open the app on the second computer as well and everything's fine.

I think I'll have to careful not to open the app on both systems at the same time.
